ui/src/components/DeleteModal/DeleteModal.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from '@testing-library/react';
+import React, { createContext } from 'react';
+import userEvent from '@testing-library/user-event';
+import { AxiosResponse } from 'axios';
+import * as axiosWrapper from '../../util/axiosCallWrapper';
+import DeleteModal from './DeleteModal';
+import { setUnifiedConfig } from '../../util/util';
+import { UnifiedConfig } from './DeleteModal.stories';
+
+jest.mock('immutability-helper', () => {
```

Comment:
It see it does not work without mocking this..
![image](https://github.com/splunk/addonfactory-ucc-generator/assets/142901247/a1674077-3ca4-411c-a7a6-a0096b4788fd)

Reply:
not sure why but Delete model does not see the mocked context, probably did something wrong, but is throwing error just in this function, as it tries to convert undefined (this.context.rowData) into object,
replaced it with general mock `jest.mock('immutability-helper');`